# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/toolkit.mozbuild
# Commit: 85fd9fc9cffa
# Full Hash: 85fd9fc9cffa98039328796d30a2d10386283b3e
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 2
# Description:
#   Bug 1742437 - Move module loader files into a central folder; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137532
# ==============================================================================

diff -r 7ecc4f9b20ad -r 85fd9fc9cffa toolkit/toolkit.mozbuild
--- a/toolkit/toolkit.mozbuild	Mon Feb 21 15:39:08 2022 +0000
+++ b/toolkit/toolkit.mozbuild	Mon Feb 21 15:39:08 2022 +0000
@@ -74,6 +74,7 @@
 DIRS += [
     '/ipc',
     '/hal',
+    '/js/loader',
     '/js/xpconnect',
     '/modules/libjar',
     '/storage',
